randovania/games/dread/json_data/Artaria.txt

```diff
@@ -558,6 +558,8 @@ Extra - asset_id: collision_camera_005
   * Extra - right_shield_def: None
   > Door to White EMMI Arena
       Trivial
+  > Dock to Early Cloak Room (Water Blob)
+      Movement (Beginner) and Lay Bomb
```

Comment:
I put this down as intermediate movement (you dont actually need anything or a wall jump to do it). It's harder than it seems